Week2Project/src/test/java/test/asserts/CrocodileAsserts.java

```diff
@@ -8,58 +9,63 @@
 
     public SoftAssert softAssert = new SoftAssert();
 
-    public void assertCreateNewCrocodile(CreateCrocodileResponse createCrocodileResponse, CreateCrocodileRequest createCrocodileRequest) {
-        softAssert.assertEquals(createCrocodileResponse.getName(), createCrocodileRequest.getName(), "Name didn't match");
-        softAssert.assertEquals(createCrocodileResponse.getDateOfBirth(), createCrocodileRequest.getDateOfBirth(), "Date of birth didn't match");
-        softAssert.assertEquals(createCrocodileResponse.getSex(), createCrocodileRequest.getSex(), "Sex didn't match");
+    public void assertCreateNewCrocodile(CrocodileResponse response, CrocodileRequest request) {
+        softAssert.assertEquals(response.getName(), request.getName(), "Name didn't match");
```

Comment:
proveravamo da id nije null, a mozemo i da napravimo metodu koja ce da racuna broj godina da i to uporedimo :)